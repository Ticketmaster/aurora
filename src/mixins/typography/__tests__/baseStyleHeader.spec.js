import BASE_STYLE_HEADER from "../baseStyleHeader";

describe("BASE_STYLE_HEADER", () => {
  it("should return the correct values", () => {
    expect(BASE_STYLE_HEADER).toMatchSnapshot();
  });
});
