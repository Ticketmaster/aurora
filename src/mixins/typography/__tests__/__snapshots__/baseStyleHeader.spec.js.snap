// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BASE_STYLE_HEADER should return the correct values 1`] = `
Array [
  "
  ",
  "
  margin: 2px 0;
  ",
  "
  color: ",
  [Function],
  ";
",
  " ",
  "
  ",
  [Function],
  ";
",
  ";
",
  ";
  ",
  "
  display: inline;
",
  ";
  ",
  "
  line-height: ",
  "1.25",
  ";
",
  ";
  ",
  "
  ",
  [Function],
  ";
",
  ";
",
]
`;
